docker/quarto/README.md

```diff
@@ -0,0 +1,55 @@
+# Quarto runner
+
+In this directory we define a dockerized environment to run Quarto. The image
+builds on top of the [VAST image](../../Dockerfile) and adds R, Poetry, and
+Quarto. This creates a runtime suitable for calling the VAST binary from
+notebooks.
+
+We don't publish the Quarto Docker image, so you need to build it locally.
+
+The `docker-compose.bind.yaml` overlay contains the configurations to bind mount
```

Comment:
Please also put all newly added docker-compose files into the list in `docker/README.md`.

Reply:
Oops, missed this. While filling it, I also realized that the default `quarto/docker-compose.yaml` didn't make much sense at it, so I added back the default layer in case you don't use the bind mount.